packages/compat/src/v1-addon.ts

```diff
@@ -919,3 +956,10 @@ function babelPluginAllowedInStage1(plugin: PluginItem) {
 function notColocatedTemplate(path: string) {
   return !/^\.\/components\/.*\.hbs$/.test(path);
 }
+
+const markedEmptyTree = new UnwatchedDir(process.cwd());
+const returnMarkedEmptyTree = {
```

Comment:
@ef4 I ran into an issue with this on an addon that calls `this._super` instead of `this._super.treeFor`. Currently the way we are mocking this as a POJO results them getting an error with: `Uncaught TypeError: this._super is not a function`. Obviously, this can easily be fixed by changing their call to `this._super.treeFor` but not sure how wide spread that is or if that is even something we should support.

Reply:
Ok, yeah, we can support that. It looks like `core-object` supports both, which is... ugh.

I filed https://github.com/embroider-build/embroider/issues/569